.github/scripts/version-set.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+VERSION="$1"
+PUSH=${2:-"no-push"}
+ADD_COMMIT=${3:-"add-commit"}
+
+VERSION_FILE='app/__version__.py'
+VERSION_PLACEHOLDER='__version__ =\s+"(.*)"$'
+DEV_PLACEHOLDER='_dev_version =\s+"(.*)"$'
+
+function get_version()
+{
+  local version=$(grep -P "$2" "$1" | sed -E "s/$2/\1/")
+  echo "$version"
+}
+
+function set_version()
+{
+  sed -i -E "s/$2/__version__ = \"$3\"/g" "$1"
+  echo $(get_version "$1" "$2")
+}
+
+OLD_VERSION=$(get_version "$VERSION_FILE" "$VERSION_PLACEHOLDER")
+DEV_VERSION=$(get_version "$VERSION_FILE" "$DEV_PLACEHOLDER")
+NEW_VERSION=$(echo ${VERSION:-"$DEV_VERSION"} | sed -E 's/^v([0-9].*)/\1/')
+echo "setting version from \"$OLD_VERSION\" to \"$NEW_VERSION\""
+FINAL_VERSION=$(set_version "$VERSION_FILE" "$VERSION_PLACEHOLDER" "$NEW_VERSION")
+echo "version changed to \"$FINAL_VERSION\""
+echo "VERSION=$FINAL_VERSION" >> $GITHUB_ENV
```

Comment:
Originally I think still wanted to use:
```__version__ =  _dev_version```
But looking at the new script, I don't see a major difference. The only important thing is to ensure that `_dev_version` value and `__version__ = ` (on a dev branch) are in sync: the script guarantees this after releases, but not if for some reason, we manually change the value of `_dev_version`. However, I don't see any reason why we would do that... so I feel like leaving this as is to avoid adding twisted logic to this script.